### DISCUSSION TITLE
How to import passkeys from Bitwarden into KeePassXC?

### DISCUSSION THREAD

**Discussion by hu3rror** [keepassxreboot/keepassxc#10716] [Q&A]
The current situation after import is that there are entries with only the name of the passkey, but they contain no content.

**Comment by droidmonkey** [comment]
You will need to manually copy the passkey information over at the time. There is no direct way to export passkeys from bitwarden.

**Reply by hu3rror** [reply]
OK, thank you!

**Reply by hu3rror** [reply]
Hi, but I have another question. In the passkey data exported from Bitwarden, there is a credentialId field. However, its format is in the form of a UUID like e64a25a4-3081-4bc4-baf3-426638381cf6, which differs from the default style used by KeePassXC.
